Sources/SwifterSwift/SwiftStdlib/StringExtensions.swift

```diff
@@ -586,6 +586,36 @@ public extension String {
     }
     #endif
 
+    #if canImport(Foundation)
```

Comment:
This is from other PR right?

Reply:
Yes. Do you want me to remove from this one?

Reply:
Sure :)

Reply:
Removed from this one 👍 !